projects/frontend/shared-components/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Change Log - VDK Shared Components Library (@vdk/shared)
```

Comment:
I don't think we maintain changelogs anymore. They proved hard to maintain consistently and with quality. 

We rely on good commit messages and auto-generated release notes - https://github.com/vmware/versatile-data-kit/releases 

I can see we have not updated https://github.com/vmware/versatile-data-kit/blob/main/CONTRIBUTING.md#changelog which  I will do.